### ISSUE TITLE
SQL error: unknown column deleted_at

### ISSUE THREAD

**Issue by jrmypttrsn** [laravelista/comments#94]
I just installed the package, ran migrations, added the correct traits to my models, included the view helper in my view (```@comments(['model' => $bug])```) and get the following error when loading the page:

```sql
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'comments.deleted_at' in 'where clause' (SQL: select * from `comments` where `comments`.`commentable_id` = 1 and comments`.`commentable_id` is not null and `comments`.`commentable_type` = App\Bug and `comments`.`deleted_at` is null) (View: /project_name/resources/views/vendor/comments/components/comments.blade.php) 
```

Thoughts?

**Comment by mabasic** [comment]
What version are you using? In the latest version of the migrations file you can see that soft deletes (deleted_at) are being added to the table, so that should work. Check your table and verify that you have that column. If you are upgrading from an older version of this package and you already had run the migration previously, you will have to update that table accorsing to the readme. See troubleshooting (upgrading) chapter in the readme or UPGRADE.md

**Comment by jrmypttrsn** [comment]
Using the most recent version, 3.5.1. Deleted_at column is there. I even created a comment directly in the database and still get the same error.

**Comment by mabasic** [comment]
I've tested this and it works correctly on a fresh install. Check your database connection to ensure that you are using the correct database and that your database table comments has column `deleted_at`.

**Comment by WBDC** [comment]
@mabasic I had v3.1 and updated my composer to ^3.4. This didn't create additional migration file (to add the soft delete field). I've tried to publish the migration files, but obviously this didn't published the changes, nor created new migration file. 
I had to change the batch number in the database migration and roll it back. I didn't have any comments so I was able to do this though... 
After adding the --force publish command option  added the required (->softDeletes()) fields to the migration file. 
I was able to fix my problem this way, but others might have comments already... My approach was sufficient enough , but I didn't see any upgrade in your readme.
